docker/azure-function/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Using specific version of docker image, due to issues at upstream. Once
+# https://github.com/Azure/azure-functions-docker/issues/281 is resolved, we can update
+# this.
+FROM mcr.microsoft.com/azure-functions/python:3.0.13901-python3.7
```

Comment:
this is only for python 3.7 right? what if the saved bundle is created with a different version?

Reply:
You may want to do something like this: https://github.com/bentoml/BentoML/pull/822#discussion_r443270834

And in the deployment operator code, it can choose which image to use based on the saved bundle's python version